custom/src/test/java/co/elastic/otel/ElasticAutoConfigurationCustomizerProviderTest.java

```diff
@@ -88,6 +95,29 @@ void customizeMetricTemporalityPreference() {
     assertThat(value).isEqualTo("LOWMEMORY");
   }
 
+  @Test
+  void verifyEffectiveDefaultTemporality() {
```

Comment:
optional: since the E in EDOT is Elastic, worth a javadoc comment mentioning that these defaults are tuned to what EDOT collector supports? (or which are not)

Reply:
I've renamed the test method to clarify that we override the default.
For the reasoning on why we do this, there's a [section in the docs](https://elastic.github.io/opentelemetry/edot-sdks/java/features.html#metric-temporality). We don't do this just for histograms, but also because handling cumulative counters is still not great, though it's getting better with ES|QL.